volatility3/framework/symbols/windows/extensions/__init__.py

```diff
@@ -651,7 +651,7 @@ def get_is_wow64(self):
         except AttributeError:
             return False
 
-        return value != 0 and value != None
+        return not not value
```

Comment:
This doesn't seem to make things clearer...  I agree it should be is not None, but I'd prefer `if value` to `if not not value`, if you really feels it needs condensing that much...

Reply:
Done. Hope it works now